e2e/test_search_immunization.py

```diff
@@ -73,3 +73,16 @@ def test_search_ignore_deleted(self):
         self.assertTrue(mmr1["id"] in resource_ids)
         self.assertTrue(mmr2["id"] in resource_ids)
         self.assertTrue(deleted_mmr["id"] not in resource_ids)
+
+    def test_search_reject_tbc(self):
+        # Given patient has a vaccine with no NHS number
+        imms = create_an_imms_obj(str(uuid.uuid4()), "TBC", mmr_code)
+        del imms["contained"][1]["identifier"][0]["value"]
+        imms["contained"][1]["identifier"][0]["extension"][0]["valueCodeableConcept"]["coding"][0]["code"] = "04"
+        self.store_records(imms)
+
+        # When
+        response = self.default_imms_api.search_immunizations("TBC", "MMR")
+
+        # Then
+        self.assertEqual(response.status_code, 500, response.text)
```

Comment:
Why 500, should this not be 404?

Reply:
This test has been removed and moved into validation tests